src/defaultSettings.js

```diff
@@ -94,16 +103,9 @@ export const controlInputs = [
     },
     { type: 'number', label: 'Lower Limit', option: 'y.domain[0]', require: true },
     { type: 'number', label: 'Upper Limit', option: 'y.domain[1]', require: true },
-    {
```

Comment:
Have we confirmed with Jack that this is ok to remove? Seems like it would be easy enough to keep. Maybe just refactor using the same workflow as `removeUnscheduledVisits.js` below.

Reply:
On further thought, I think the "hide visits with no data" control is worth keeping unless there is a compelling reason not to do so ...

Reply:
Yeah, on second thought it's nice to have.